netops_demo/golang/README.md

```diff
@@ -33,368 +33,106 @@ nuctl deploy \
-            1.5021898100584443,
-            1.6458255892499305,
-            2.4210833300639285,
-            0.911913324650527
-        ],
-        "alerts": [
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "High Packet Loss (2.4210833300639285) above threshold (2.0)",
-            ""
-        ],
-        "is_error": [
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0
-        ]
-    },
-    "latency": {
-        "timestamps": [
-            1533733834000,
-            1533733834050,
-            1533733834100,
-            1533733834150,
-            1533733834200,
-            1533733834250,
-            1533733834300,
-            1533733834350,
-            1533733834400,
-            1533733834450,
-            1533733834500,
-            1533733834550,
-            1533733834600,
-            1533733834650,
-            1533733834700,
-            1533733834750,
-            1533733834800,
-            1533733834850,
-            1533733834900,
-            1533733834950
-        ],
-        "values": [
-            5.398297647696318,
-            3.8738349820215303,
-            3.6242822658204896,
-            6.436707909468402,
-            5.166855956117543,
-            6.382151916625901,
-            5.210301543972159,
-            6.946586377924426,
-            5.125703929135469,
-            5.175648172419271,
-            2.809223009632173,
-            5.922469923022994,
-            10,
-            5.5015345911571325,
-            4.142279789761261,
-            4.543633483503898,
-            3.379849311282091,
-            4.691857946122445,
-            4.5161436688962215,
-            4.395570604095654
-        ],
-        "alerts": [
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "High Latency (10) above threshold (7.0)",
-            "",
-            "",
-            "",
-            "",
-            "",
-            "",
-            ""
-        ],
-        "is_error": [
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            0,
-            1,
-            1,
-            1,
-            1,
-            1,
-            1,
-            1,
-            1,
-            1
-        ]
-    }
-}
+[
```

Comment:
What is the purpose of the outer list?

Reply:
_generate_batch now returns:
[{Company1-location1-device1 metrics}, ..., {CompanyN-locationN-deviceN metrics}]

_generate has a loop with responses.append(response)

So we end up with nested array